app/src/main/kotlin/batect/docker/DockerContainerCreationRequestFactory.kt

```diff
@@ -34,8 +34,14 @@ class DockerContainerCreationRequestFactory(
         userAndGroup: UserAndGroup?,
         terminalType: String?,
         useTTY: Boolean,
-        attachStdin: Boolean
+        attachStdin: Boolean,
+        disablePortMappings: Boolean
```

Comment:
We probably don't need to pass this in as a parameter here - `DockerContainerCreationRequestFactory` could be injected with the `CommandLineOptions` instance. 

This would also remove the need to add another property to `RunOptions`, which already has a few too many properties (I'd really like to remove `propagateProxyEnvironmentVariables` and `imageOverrides` at some point, as they don't change per task).